src/main/java/edu/ksu/canvas/requestOptions/SubmissionOptions.java

```diff
@@ -0,0 +1,56 @@
+package edu.ksu.canvas.requestOptions;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Submission options to submit an assignment on behalf of a user.
```

Comment:
While it can submit on behalf of a student, it only does that if you set the optional `user_id`